test/allEqual.js

```diff
@@ -0,0 +1,15 @@
+import { assert } from 'chai';
+
+import * as RA from '../src/index';
+
+describe('allEqual', function() {
+  it('returns true when all items are equal', function() {
+    assert.equal(RA.allEqual([4, 4, 4, 4]), true);
+  });
+  it('returns true when all items are deeply equal', function() {
```

Comment:
Use `should` convention in tests. Can be seen here: https://github.com/char0n/ramda-adjunct/blob/f7d7f7c454c2e49806b3fc0e1e521577a38ea544/test/argsPass.js#L32